cmd/service.go

```diff
@@ -101,7 +101,9 @@ A full list of service types that can be deployed are available using: 'odo cata
 		}
 		err = svc.CreateService(client, serviceName, serviceType, plan, parameters, applicationName)
 		checkError(err, "")
-		fmt.Printf("Service '%s' was created.\n", serviceName)
+		fmt.Printf(`Service '%s' was created.
+Progress of the provisioning will not be reported and might take a long time
```

Comment:
`Status` of the service can be seen via `odo service list`

for example,

```
$ odo service list
NAME                         TYPE                         STATUS
cakephp-mysql-persistent     cakephp-mysql-persistent     ProvisionRequestInFlight
```

Once it is provisioned,

```
odo service list
NAME                         TYPE                         STATUS
cakephp-mysql-persistent     cakephp-mysql-persistent     Provisioning
```

Reply:
Good idea, I will add the command to the output

Reply:
Fixed